input/blog/2020-11-05-cake-v1.0.0-rc0001-released.md

```diff
@@ -0,0 +1,234 @@
+
+RunTarget("Build");
+```
+
+### Implicit bootstrapping
+
+Boostrapping of Cake modules are now implicit, which means that you no longer will need to execute Cake twice to download and initialize modules, just add the module directive to your script and this will be handled transparently for you.
+
+```c#
+#module nuget:?package=Cake.DotNetTool.Module&version=0.4.0
+```
+
+### Rewritten CLI
+
+We're now utilizing [Spectre.Cli](https://www.nuget.org/packages/Spectre.Cli/) for a more robust, flexible and future prof handling of
+
+Full details of everything that was included in this release can be seen below.
+
+### Frosting incorporated into main rep
+
+Cake Frosting is now incorporated in same GitHub repository as ( [cake-build/cake](https://github.com/cake-build/cake/) ) all Cake other Cake runners, this means it'll be released and supported with the same cadence as them.
+
+### Supported platforms
+
+| Runner | .NET 5 | .NET 4.6.1 | .NET Core 3.1 | .NET Core 3.0 | .NET Core 2.1 | .NET Core 2.0 | .NET Standard 2.0 | Mono |
+|--------|--------|------------|---------------|---------------|---------------|---------------|-------------------|------|
+| [Cake.Tool](https://www.nuget.org/packages/Cake.Tool/1.0.0-rc0001)           | ✔  | ❌ | ✔  | ✔  | ✔  | ❌ | ❌ | ❌ |
+| [Cake.Frosting](https://www.nuget.org/packages/Cake.Frosting/1.0.0-rc0001)   | ✔  | ✔  | ✔  | ✔  | ✔  | ✔  | ✔  | ✔  |
+| [Cake](https://www.nuget.org/packages/Cake/1.0.0-rc0001)                     | ❌ | ✔  | ❌ | ❌ | ❌ | ❌ | ❌ | ✔  |
+| [Cake.CoreCLR](https://www.nuget.org/packages/Cake.CoreCLR/1.0.0-rc0001)     | ❌ | ❌ | ❌ | ❌ | ❌  | ✔ | ❌ | ❌ |
+
+### Supported operating systems
+
+| Runner | macOS | Linux | Windows |
+| ------ | ----- | ----- | ------- |
+| [Cake.Tool](https://www.nuget.org/packages/Cake.Tool/1.0.0-rc0001)           | ✔  | ✔  | ✔  |
+| [Cake.Frosting](https://www.nuget.org/packages/Cake.Frosting/1.0.0-rc0001)   | ✔  | ✔  | ✔  |
+| [Cake](https://www.nuget.org/packages/Cake/1.0.0-rc0001)                     | ✔  | ✔  | ✔  |
+| [Cake.CoreCLR](https://www.nuget.org/packages/Cake.CoreCLR/1.0.0-rc0001)     | ✔  | ✔  | ✔  |
+
+### Supported CI Systems
+
+| Runner | AppVeyor | Azure Pipelines | Bamboo | Bitbucket Pipelines | Bitrise | Continua CI | GitHub Actions | GitLab CI | GoCD | Jenkins | MyGet | TeamCity | TravisCI |
+| - | - | - | - | - | - | - | - | - | - | - | - | - | - |
+| [Cake.Tool](https://www.nuget.org/packages/Cake.Tool/1.0.0-rc0001)           | ✔ | ✔ | ✔ | ✔ | ✔ | ✔ | ✔ | ✔ | ✔ | ✔ | ✔ | ✔ | ✔ |
+| [Cake.Frosting](https://www.nuget.org/packages/Cake.Frosting/1.0.0-rc0001)   | ✔ | ✔ | ✔ | ✔ | ✔ | ✔ | ✔ | ✔ | ✔ | ✔ | ✔ | ✔ | ✔ |
+| [Cake](https://www.nuget.org/packages/Cake/1.0.0-rc0001)                     | ✔ | ✔ | ✔ | ✔ | ✔ | ✔ | ✔ | ✔ | ✔ | ✔ | ✔ | ✔ | ✔ |
+| [Cake.CoreCLR](https://www.nuget.org/packages/Cake.CoreCLR/1.0.0-rc0001)     | ✔ | ✔ | ✔ | ✔ | ✔ | ✔ | ✔ | ✔ | ✔ | ✔ | ✔ | ✔ | ✔ |
```

Comment:
Use the same naming for runners as on [documentation](https://cakebuild.net/docs/running-builds/runners/) and link to the runner documentation instead of NuGet package.

Reply:
adjusted